File: ParticipantDTO.java
package me.model;

import java.io.Serializable;

public class ParticipantDTO implements Serializable {
    private int id;
    private String nume;
    private int varsta;
    private String listaProbe;

    public ParticipantDTO(int id, String nume, int varsta, String listaProbe) {
        this.id = id;
        this.nume = nume;
        this.varsta = varsta;
        this.listaProbe = listaProbe;
    }

    public int getId() {
        return id;
    }

    public String getNume() {
        return nume;
    }

    public int getVarsta() {
        return varsta;
    }

    public String getListaProbe() {
        return listaProbe;
    }
}

File: Concurs Inot C#/ParticipantDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public class ParticipantDTO
    {
        public int Id { get; }
        public String Nume { get; }
        public int Varsta { get; }
        public String ListaProbe { get; }

        public ParticipantDTO(int id, string nume, int varsta, string listaProbe)
        {
            Id = id;
            Nume = nume;
            Varsta = varsta;
            ListaProbe = listaProbe;
        }
    }
}
